libs/backend/src/scan/cast_vote_records/build_report_metadata.ts

```diff
@@ -146,7 +146,7 @@ interface BuildCastVoteRecordReportParams {
  * necessary to make sense of those CVRs. We generate this separately because
  * we want to write the CVRs to file separately, one-by-one.
  */
-export function buildCastVoteRecordReport({
+export function buildCastVoteRecordReportWithoutCastVoteRecords({
```

Comment:
Maybe call this `buildCastVoteRecordReportMetadata` to match with the file name? It takes a bit of thinking to figure out what it means to build a cast vote record report without cast vote records.

Reply:
Sounds good to me.